README.md

```diff
@@ -147,17 +147,6 @@ get fooBar() {
 }
 ```
 
-To set the type of a computed property when the Metadata Reflection API is not
```

Comment:
Why are you removing this bit? Should still apply AFAICT.

Reply:
See discussion in https://github.com/Polymer/polymer-decorators/pull/51#issuecomment-368787769

TLDR there's no reason to set the type of a computed property, because it's only used for attribute de-serialization.